servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -258,13 +265,16 @@ def _get_json(self, url, timeout, auth=None, verify=True):
             if status is AgentCheck.CRITICAL:
                 self.service_check(self.SERVICE_CHECK_NAME, status, tags=tags,
                                    message=msg)
-                raise Exception("Cannot connect to ServiceNow CMDB, please check your configuration.")
+                raise CheckException("Cannot connect to ServiceNow CMDB, please check your configuration.")
 
         if resp.encoding is None:
             resp.encoding = 'UTF8'
         try:
-            resp = yaml.safe_load(resp.text)
+            resp = yaml.safe_load(resp.text.encode("utf-8"))
         except Exception as e:
             self.log.exception(str(e))
-            raise Exception("Exception occured while parsing response with Yaml and the error is : {}".format(str(e)))
+            raise CheckException("Exception occured while parsing response and the error is : {}".format(str(e)))
+
+        if "error" in resp and resp["error"]:
```

Comment:
isn't this the same as saying `if resp.get("error"):`?

Reply:
fixed